.github/workflows/release-build.yaml

```diff
@@ -15,6 +15,8 @@ on:
         default: ''
 
 jobs:
+  run_playwright_tests:
```

Comment:
Don't you need to also add dependency?
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds

Reply:
The pipeline behaves as expected after fix: if there are some failed tests in PW tests, "Release Build" will not be triggered:
https://github.com/opendatadiscovery/odd-platform/actions/runs/3045883991/jobs/4907998093